tests/unit/drivers/yaml/mockencoder-mode1.yml

```diff
@@ -9,4 +9,4 @@ requests:
       - !EncodeDriver
         with:
           method: encode
-          granularity_range: [1, 2]
+          traversal_paths: ['r']
```

Comment:
This should be `c` instead of `r`, since beforehand, the mockencoder also encoded the chunks (e.g. `granularity_level = 1`).